ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -34,16 +39,15 @@ import scalaz.syntax.std.option._
 import scalaz.std.scalaFuture._
 import scalaz.std.map._
 import scalaz.std.option._
-import scalaz.std.set._
 import scalaz.std.tuple._
 import scalaz.std.vector._
 import scalaz.syntax.traverse._
 import scalaz.std.list._
-import scalaz.{@@, -\/, \/, \/-, Foldable, Liskov, NonEmptyList, OneAnd, Semigroup, Tag}
+import scalaz.{-\/, Foldable, Liskov, NonEmptyList, OneAnd, Tag, \/, \/-}
 import Liskov.<~<
 import com.daml.fetchcontracts.domain.ResolvedQuery
 import ResolvedQuery.Unsupported
-import com.daml.fetchcontracts.domain.ContractTypeId.{toLedgerApiValue, OptionalPkg}
+import com.daml.fetchcontracts.domain.ContractTypeId.{OptionalPkg, toLedgerApiValue}
```

Comment:
Please use `git add -p` to avoid IntelliJ spam.

Reply:
I'm not too worried about Intellij sorting these to be honest.

Reply:
The problem is that it's not stable. The next change will sort them a different way, and then the next change, &c.